README.md

```diff
@@ -0,0 +1,46 @@
+# hexabot-nlu
+## Overview
+This repository contains a Natural Language Understanding (NLU) engine built using Ludwig AI, a high-level machine learning framework. Ludwig simplifies the process of training and serving machine learning models, enabling rapid development without extensive coding.
+## Features
```

Comment:
Create some line spacing + add some sentences to the commands

I think results, predictions, ... should be outside of "src" folder